core/api/src/services/notifications/convert.ts

```diff
@@ -65,18 +63,18 @@ export const notificationCategoryToGrpcNotificationCategory = (
   category: NotificationCategory,
 ): GrpcNotificationCategory => {
   switch (category) {
-    case GaloyNotificationCategories.Payments:
+    case NotificationCategory.Payments:
       return GrpcNotificationCategory.PAYMENTS
-    case "Circles":
+    case NotificationCategory.Circles:
       return GrpcNotificationCategory.CIRCLES
-    case "Price":
+    case NotificationCategory.Price:
       return GrpcNotificationCategory.PRICE
-    case GaloyNotificationCategories.AdminNotification:
+    case NotificationCategory.AdminNotification:
       return GrpcNotificationCategory.ADMIN_NOTIFICATION
-    case GaloyNotificationCategories.Balance:
+    case NotificationCategory.Balance:
       return GrpcNotificationCategory.BALANCE
-    default:
-      throw new Error(`Not implemented: ${category}`)
+    case NotificationCategory.Marketing:
+      return GrpcNotificationCategory.MARKETING
```

Comment:
it is better to keep the default returning an error.. specially if new items are added to the categories

Reply:
Will add a default